python/tk_houdini_renderman/farm_dialog.py

```diff
@@ -171,6 +178,8 @@ def __submit_to_farm(self):
             "Name=" + submission_name,
             "Department=3D",
             "EnvironmentKeyValue0 = RENDER_ENGINE = RenderMan",
+            "PostTaskScript="
```

Comment:
I think this will reference a path to the file on the local computer that submits the job, so the Deadline farm might not be able to access this python script to run. Have you checked if this runs properly on the farm when computers other than the submitter are running the job?